docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -245,12 +259,87 @@ To update an existing network with a new `blockperiodseconds`:
 1. To verify the changes after the transition block, call
    [`ibft_getValidatorsByBlockNumber`](../../../Reference/API-Methods.md#ibft_getvalidatorsbyblocknumber), specifying `latest`.
 
-!!! caution
+### Update block rewards on an existing network deployment
 
-    Do not specify a transition block in the past.
-    Specifying a transition block in the past could result in unexpected behavior, such as causing
-    the network to fork.
-{!global/Config-Options.md!}
+To update an existing network with a new `blockreward`:
+
+1. Stop all nodes in the network.
+2. In the [genesis file](#genesis-file), add the `transitions` configuration item where:
+
+    * `<FutureBlockNumber>` is the upcoming block at which to change `blockreward`.
+    * `<NewValue>` is the updated value for `blockreward`.
+
+    !!! example "Transitions configuration"
+
+        === "Syntax"
+
+            ```bash
+            {
+              "config": {
+                 ...
+                 "ibft2": {
+                   "blockperiodseconds": 2,
+                   "epochlength": 30000,
+                   "requesttimeoutseconds": 4
```

Comment:
Maybe should have an initial blockreward value here to show what its being changed in the transition below?

Reply:
It defaults to zero if you don't specify it in the initial config, so the example is still correct. If the intention is to go from 0 -> +ve reward then IMO we should make that explicit either in the preamble or by specifying initial config.